docs/deploying.md

```diff
@@ -31,64 +26,126 @@ image. You push an image to, and then pull the same image from, this registry.
 This a good exercise for understanding the basic interactions a client has with
 a local registry.
 
-1. Install Docker.
+### Setting up with Docker Machine
```

Comment:
I know this PR is closed, but unless you intend to support Docker Machine in this project, I recommend that we avoid this. This should be for setting up the registry. Not nginx, not apache, not docker machine, not to advertise the hub, not to teach people bash.